README.md

```diff
@@ -173,3 +173,19 @@ Use this options to provide a specific space name of the registry to publish the
 #### `--verbose`
 
 To print more logs when using tool (useful for debug).
+
+### Recommended workflow
+
+#### Day to day
+
+- Development is done on feature branches that are merged into `master`,  once they are complete.
+- Every time someone commits on `master`, a new archive is created an uploaded on Downcloud and it is then deployed on a test instance with Rundeck.
+
+#### Release workflow
+
+- A new branch is created from the current state of `master`. Let's say we want to deploy version `1.0.0` of the app.
+- The only type of commits allowed on this branch are bug fixes.
+- Every commit is tagged with a prerelease version number, eg. `1.0.0-beta.1`, `1.0.0-beta.2`, etc…
```

Comment:
Actually, not every commit is tagged. A tag is generated every time we want to publish a new beta. A new beta may contains several fixes and several commits.